dds/DCPS/Serializer.cpp

```diff
@@ -62,36 +62,52 @@ Serializer::swapcpy(char* to, const char* from, size_t n)
   switch (n) {               // 2   4   8   16
   case 16:
     to[ 15] = from[ n - 16]; // x   x   x    0
+    // fallthrough
```

Comment:
GCC is changing its output of warnings based on a comment?  I guess that's good??